praw/models/reddit/subreddit.py

```diff
@@ -202,6 +202,20 @@ def _subreddit_list(subreddit, other_subreddits):
             return ",".join([str(subreddit)] + [str(x) for x in other_subreddits])
         return str(subreddit)
 
+    @staticmethod
+    def _validate_gallery(images):
+        for image in images:
+            image_path = image.get("image_path", "")
+            if not isfile(image_path):
+                if image_path:
```

Comment:
The check for a blank string should come before `isfile` since it's a cheaper test to perform.